src/Components/SalesforceWrapper.tsx

```diff
@@ -33,6 +57,7 @@ export const SalesforceWrapper: React.FC = () => {
     )}/embeddedservice/5.0/esw.min.js`,
     removeOnUnmount: true,
     onReady: () => {
+      window.embedded_svc.settings.defaultMinimizedText = "Chat"
```

Comment:
Taking the chance to handle this here, but it is actually part of [NX-3562](https://artsyproduct.atlassian.net/browse/NX-3562)

(I checked with Allie, and there is no problem is releasing the `Chat with expert` > `Chat` changes in different dates for the different apps)